src/equation_of_state.jl

```diff
@@ -1,15 +1,26 @@
 """
-    LinearEquationOfState
+    LinearEquationOfState{T<:AbstractFloat} <: AbstractEquationOfState
```

Comment:
Couldn't `T` be an `Int`? Nothing wrong with that right? Ie, the constants are `0`.